"npm start" often returns an error at the end

In the README.md, this is written to use `npm start --` (instead of ` ./node_modules/.bin/ts-node ./src/cli.ts`). But this does not work as good. For example:

```
$ npm start 

> mwoffliner@1.9.3 start /dev/shm/mwoffliner
> ts-node ./src/cli.ts

Create a fancy HTML dump of a Mediawiki instance in a directory or as a ZIM file
  Usage: cli.ts
  Example, as a system tool:
  mwoffliner --mwUrl=https://en.wikipedia.org/ --adminEmail=foo@bar.net
  Or, as a node script:
  node mwoffliner.js --mwUrl=https://en.wikipedia.org/ --adminEmail=foo@bar.net
  Or, as a npm script: '
  npm run mwoffliner -- --mwUrl=https://en.wikipedia.org/
  --adminEmail=foo@bar.net

Options:
  --version                  Show version number                       [boolean]
  --help                     Show help                                 [boolean]
  --mwUrl                    Mediawiki base URL.                      [required]
  --adminEmail               Email of the mwoffliner user which will be put in
                             the HTTP user-agent string               [required]
  --articleList              File with one title (in UTF8) per line. This can be
                             a local path or an HTTP(S) url
  --cacheDirectory           Directory where files are permanently cached
  --customZimFavicon         Use this option to give a path to a PNG favicon, it
                             will be used in place of the Mediawiki logo. This
                             can be a local path or an HTTP(S) url
  --customZimTitle           Allow to configure a custom ZIM file title.
  --customZimDescription     Allow to configure a custom ZIM file description.
  --customZimTags            Allow to configure custom ZIM file tags (semi-colon
                             separated).
  --customMainPage           Allow to configure a custom page as welcome page.
  --filenamePrefix           For the part of the ZIM filename which is before
                             the date part.
  --format                   Each --format=... will cause a new file to be
                             created, acceptable options are:
                             "nopic,novid,nopdf,nozim,nodet"
                             e.g. "--format=nopic --format=novid,nopdf" will
                             create two ZIM files
  --keepEmptyParagraphs      Keep all paragraphs, even empty ones.
  --keepHtml                 If ZIM built, keep the temporary HTML directory
  --mwWikiPath               Mediawiki wiki base path (per default "/wiki/")
  --mwApiPath                Mediawiki API path (per default "/w/api.php")
  --mwModulePath             Mediawiki module load path (per default
                             "/w/load.php")
  --mwDomain                 Mediawiki user domain (thought for private wikis)
  --mwUsername               Mediawiki username (thought for private wikis)
  --mwPassword               Mediawiki user password (thought for private wikis)
  --minifyHtml               Try to reduce the size of the HTML
  --outputDirectory          Directory to write the downloaded content
  --publisher                ZIM publisher meta data, per default 'Kiwix'
  --redis                    Redis configuration (https://github.com/NodeRedis/n
                             ode_redis#rediscreateclient)
  --requestTimeout           Request timeout (in seconds)
  --resume                   Do not overwrite if ZIM file already created
  --useCache                 Cache all assets and HTML
  --skipCacheCleaning        Do not search for old/outdated files in the cache
  --speed                    Multiplicator for the number of parallel HTTP
                             requests on Parsoid backend (per default the number
                             of CPU cores). The default value is 1.
  --verbose                  Print debug information to the stdout
  --withoutZimFullTextIndex  Don't include a fulltext search index to the ZIM
  --addNamespaces            Force addional namespace (comma separated numbers)
  --getCategories            [WIP] Download category pages
  --noLocalParserFallback    Don't fall back to a local MCS or Parsoid, only use
                             remote APIs

Missing required arguments: mwUrl, adminEmail
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mwoffliner@1.9.3 start: `ts-node ./src/cli.ts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mwoffliner@1.9.3 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kelson/.npm/_logs/2019-07-06T13_49_34_862Z-debug.log
```

this part of the message should not be here:
```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mwoffliner@1.9.3 start: `ts-node ./src/cli.ts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mwoffliner@1.9.3 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kelson/.npm/_logs/2019-07-06T13_49_34_862Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -85,13 +85,21 @@ mwoffliner.execute(parameters); // returns a Promise
 
 ## Development
 > Please see [CONTRIBUTING.md](./CONTRIBUTING.md)
+
+To setup mwoffliner locally:
 ```bash
-# Setup
 git clone https://github.com/openzim/mwoffliner.git
 cd mwoffliner
 npm i
 
-# Running
+To run it (this is only an example):
+```bash
+./node_modules/.bin/ts-node ./src/cli.ts --mwUrl=https://bm.wikipedia.org --adminEmail=XXX
+```
+
+or
+
+```bash
 npm start -- --mwUrl=https://bm.wikipedia.org --adminEmail=XXX
 ```
 

